docs/installation-deb.md

```diff
@@ -4,11 +4,9 @@
 
 We'll go over these in more detail below.
 
-* Cassandra. We run and recommend 3.0.8 .
+* Cassandra. We run and recommend 3.X .
```

Comment:
yep you are correct. I wrongly assumed all 3.X versions had twcs, but it was not added till 3.8